lib/remap-functions/src/is_nullish.rs

```diff
@@ -0,0 +1,161 @@
+use lazy_static::lazy_static;
+use regex::Regex;
+use remap::prelude::*;
+
+lazy_static! {
+    // This pattern considers each of these as nullish:
+    // * Whitespace-only strings of any length
+    // * Newline (\n)
+    // * Carriage return (\r)
+    // * Single dash (-)
+    static ref NULLISH_STRING_PATTERN: Regex = Regex::new(r"^(\s*|\\n|\\r|\-)$").unwrap();
```

Comment:
It might make sense to use Rust's [`is_whitespace`](https://doc.rust-lang.org/std/primitive.char.html#method.is_whitespace), since that captures more whitespace-like characters as defined by the UTF-8 DB.

See https://play.rust-lang.org/?gist=d8a026bfad5d5a83dfea9ad0034ba19a